frontend/src/components/Combobox.tsx

```diff
@@ -20,6 +20,7 @@ interface ComboboxProps<T> extends FormikCompatible<Maybe<T>> {
   isLoading?: boolean;
   styles?: StylesConfig<SelectOption, false, GroupTypeBase<SelectOption>>;
   isDisabled?: boolean;
+  optionsFilter?: (val: T[]) => T[] | undefined;
```

Comment:
are you sure you need both `?:` and `| undefined`? I don't think so

Reply:
It is not `((val: T[]) => T[]) | undefined`